README.rst

```diff
@@ -53,18 +73,23 @@ If you want to contribute your data, perform the following steps:
 
 This will create a set of log files in the "vehicle_fingerprint" folder.
 Before sending the data to anyone please **check for any personal data**.
+
 The following attributes should be replaced with default values:
```

Comment:
The vin, lat, lon, heading and license plate are automatically anomized by the code -> https://github.com/bimmerconnected/bimmer_connected/blob/184797cb4dfa15ead4b3bf0107b2bdacf95a0abe/bimmer_connected/account.py#L194.
So only need to check if there is other data they don't want to be send, like for instance the dealer info.

Reply:
Clarified this in the readme :)